demo/admin/src/products/ProductForm.cometGen.ts

```diff
@@ -0,0 +1,68 @@
+import { FormConfig } from "@src/generator/generator";
+import { GQLProduct } from "@src/graphql.generated";
+
+export const ProductForm: FormConfig<GQLProduct> = {
+    type: "form",
+    gqlType: "Product",
+    fragmentName: "ProductFormDetails", // configurable as it must be unique across project
+    fields: [
+        {
+            type: "text",
+            name: "title",
+            label: "Titel", // default is generated from name (camelCaseToHumanReadable)
```

Comment:
This is the default message, right? Is this enough, or do we need to be more flexible here (for instance, allow `ReactNode`)?

Reply:
Probably it's better to allow `ReactNode` too. I think I saw some screen designs where a helper icon was rendered next to the label. That wouldn't be possible here.

Also maybe setting a custom ID for the `FormattedMessage` though I'm not sure if that's ever requested (maybe to avoid duplicates?)

Reply:
Allowing ReactNode isn't easily possible, as we generate code based on the config in this file.